querydsl-core/src/main/java/com/querydsl/core/util/JavaSyntaxUtils.java

```diff
@@ -27,63 +25,8 @@
 
     private JavaSyntaxUtils() { }
```

Comment:
Shall we then deprecate it?
Then the next cleanup with the next major release it will pop up.